src/extension.ts

```diff
@@ -101,7 +101,7 @@ export function activate(context: vscode.ExtensionContext) {
       );
 
       lspClient.onReady().then(() => {
-        record(TelemetryActions.componentAnalysisTriggered);
+        record(TelemetryActions.componentAnalysisTriggered, {depFileName: vscode.window.activeTextEditor.document.uri.fsPath.split('/').reverse()[0]});

```

Comment:
tracking is not a correct place to track ca triggered. tracking must be inside lspClient.onNotification('caNotification', respData => {

Also you can use `path.basename('/full/path')` to get file name.

Reply:
Also please add another event for caError.